components/esp_lvgl_port/README.md

```diff
@@ -0,0 +1,89 @@
+# LVGL ESP Portation
+
+This component should helps with using LVGL with Espressif's LCD and touch drivers. It can be used with any project with LCD display. 
+
+## Features
+* Initialization of the LVGL
+    * Create task and timer
+    * Handle rotating
+* Add/remove display (using [`esp_lcd`](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/peripherals/lcd.html))
+* Add/remove touch input (using [`esp_lcd_touch`](https://github.com/espressif/esp-bsp/tree/master/components/lcd_touch))
+
+## Usage
+
+### Initialization
+```
+    esp_err_t err = lvgl_port_init();
+```
+
+### Add screen
+
+Add an LCD screen to the LVGL. It can be called more times for adding more LCD screens. 
+```
+    /* LCD driver initialization */
+    ...
+    esp_lcd_panel_handle_t lcd_panel_handle;
+    esp_err_t err = esp_lcd_new_panel_ra8875(io_handle, &panel_config, &lcd_panel_handle);
+
+    /* Add LCD screen */
+    const lvgl_port_display_cfg_t disp_cfg = {
+        .handle = lcd_panel_handle,
+        .buffer_size = DISP_WIDTH*DISP_HEIGHT,
+        .double_buffer = true,
+        .hres = DISP_WIDTH,
+        .vres = DISP_HEIGHT,
+        .monochrome = false,
+        /* Rotation values must be same as used in esp_lcd for initial settings of the screen */
+        .rotation = {
+            .swap_xy = false,
+            .mirror_x = false,
+            .mirror_y = false,
+        }
+    };
+    lv_disp_t * disp = lvgl_port_add_disp(&disp_cfg);
+```
+
+**Note 1:** The component calls `esp_lcd_panel_draw_bitmap` API for send data to the screen. There must be called `lvgl_port_flush_ready(disp)` after each transaction to display. The best way is to use `on_color_trans_done` callback from `esp_lcd` IO config structure.
```

Comment:
'Flush ready' should be handled inside of the lvgl_port.
I can see that currently this is hardly possible now... we talk about it later

Reply:
I added note into readme. We need some example too.